dolphinscheduler-ui/src/views/projects/workflow/definition/components/use-form.ts

```diff
@@ -95,7 +95,7 @@ export const useForm = () => {
     timingFormRef: ref(),
     timingForm: {
       startEndTime: [
-        new Date(year, month, day),
+        new Date(year, month, day + 1),
```

Comment:
Just curious, why did u do this here?

Reply:
It will return warning message if setting start time earlier than current time, so default time should be valid.